src/Libraries/RevitNodesUI/RevitNodesUI.csproj

```diff
@@ -39,6 +39,10 @@
       <HintPath>$(PACKAGESPATH)\DynamoVisualProgramming.Core\lib\net45\DynamoUtilities.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Newtonsoft.Json, Version=8.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\packages\Newtonsoft.Json.8.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
```

Comment:
you don't want to include the version number here - it should just be
`Newtonsoft.Json`

Reply:
@mjkkirschner I think the path need to be specific, otherwise dll will not be found. Keith was referring to other project we already have I guess, like DynamoCore. You can check DynamoCore.csproj as well

Reply:
look in your packages folder:
![screen shot 2017-09-06 at 2 39 50 pm](https://user-images.githubusercontent.com/508936/30128598-45a2eec8-9311-11e7-84cd-7bbd92f54eeb.png)

the whole point of this wacky packages sym link thing is so that the folders do not have version numbers in them.

Reply:
Yup you are right. DynamoCore is not equivalent to DynamoRevit project in this case.

Reply:
another option here is that we depend on the version of nuget that dynamo core pulls instead of adding another direct reference to it. @alfarok thoughts on that?

Reply:
@QilongTang for some reason I have 2 versions in my package folder and assumed because the batch file pulled 8.0.3 that was the version I should reference